.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,10 +18,13 @@
 - [ ] My name is on the [list of contributors](https://github.com/glotzerlab/signac/blob/master/contributors.yaml).
 - [ ] My code follows the [code style guideline](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md#code-style) of this project.
 - [ ] The changes introduced by this pull request are covered by existing or newly introduced tests.
-
-If necessary:
-- [ ] I have updated the API documentation as part of the package doc-strings.
-- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
+- [ ] I have determined whether the [framework documentation](https://docs.signac.io/) requires an update (see guidance below). If so, I have created a separate pull request to update it on [signac-docs](https://github.com/glotzerlab/signac-docs) at this link: glotzerlab/signac-docs/pull#**XXX**
```

Comment:
When this text is rendered, the “guidance below” will be invisible in a comment. Readers will not know what this guidance says, only the author of the PR. I think that is potentially confusing. However, we also want to minimize the amount of boilerplate text in each PR. I might suggest a link to the CONTRIBUTING document, and include the description there.

We might also want to unify some of our CONTRIBUTING document with the developer guide in our docs… a single source of info might be better?